public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, ReactElement, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { PanelMenuItem } from '@grafana/data';
+
+import { DashboardModel, PanelModel } from '../../state';
+import { StoreState } from '../../../../types';
+import { getPanelMenu } from '../../utils/getPanelMenu';
+
+interface Api {
```

Comment:
```suggestion
interface PanelHeaderMenuProviderApi {
```

to keep this consistent with the other naming you have in PanelHeaderMenuTrigger

Reply:
Will fix

Reply:
Fixed!